docs/book/v3/standard-filters.md

```diff
@@ -644,29 +644,40 @@ which accepts an array of all options.
 
 ## DateTimeFormatter
 
-The filter normalize a datetime string to match the filter specified format
+This filter formats either a `DateTimeInterface` object, a string, or integer that `DateTime` will understand to a date
+and/or time string in the configured format.
 
 ### Supported Options
 
 The following options are supported for `Laminas\Filter\DateTimeFormatter`
 
-- `format`: a valid date format to use when formatting a string
-- `timezone` : a valid timezone to be used when handling the date
+- `format`: a valid [date format](https://www.php.net/manual/datetime.format.php) to use when formatting a string, for example `l jS F Y`. This option defaults to `DateTimeInterface::ATOM` when unspecified
+- `timezone` : The default timezone to apply when converting a string or integer argument to a `DateTime` instance. This option falls back to the system timezone when unspecified
 
 ### Basic Usage
 
+#### Without any options
```

Comment:
```suggestion
#### Without Any Options
```

Titles should be title case 👍

Reply:
there is actually a Website which Formats any text. 
got that from @froschdesign a few years back: https://capitalizemytitle.com/style/Chicago/